probe/src/flash/flasher.rs

```diff
@@ -0,0 +1,583 @@
+use crate::debug_probe::DebugProbeError;
+use coresight::access_ports::AccessPortError;
+use ::memory::MI;
+use crate::session::Session;
+
+use super::*;
+
+const ANALYZER: [u32; 49] = [
```

Comment:
What does the analyzer do?

Reply:
Later on it will analyze the flash contents to not flash the things already matching in flash with potential speed gains :)